conda/recipes/ucxx/recipe.yaml

```
@@ -0,0 +1,323 @@
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - cmake ${{ cmake_version }}
+        - ${{ compiler("c") }}
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
+      host:
+        - ucx
+        - cuda-version =${{ cuda_version }}
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - librmm
+      run:
+        - if: cuda_major == "11"
+          then: cudatoolkit
+        - ${{ pin_compatible("cuda-version", upper_bound="x", lower_bound="x") }}
+        - ucx >=1.15.0,<1.19.0
+        - ${{ pin_compatible("librmm", upper_bound="x.x") }}
+      run_exports:
+        - ${{ pin_subpackage("libucxx", upper_bound="x.x") }}
+      ignore_run_exports:
+        from_package:
+          - cuda-cudart-dev
+          - librmm
+          - python
+          - ucx
+        by_name:
+          - cuda-version
+          - if: cuda_major == "11"
+            then: cudatoolkit
+    tests:
+      - package_contents:
+          files:
+            - include/ucxx/address.h
+            - include/ucxx/api.h
+            - include/ucxx/buffer.h
+            - include/ucxx/component.h
+            - include/ucxx/config.h
+            - include/ucxx/constructors.h
+            - include/ucxx/context.h
+            - include/ucxx/delayed_submission.h
+            - include/ucxx/endpoint.h
+            - include/ucxx/exception.h
+            - include/ucxx/future.h
+            - include/ucxx/header.h
+            - include/ucxx/inflight_requests.h
+            - include/ucxx/listener.h
+            - include/ucxx/log.h
+            - include/ucxx/notifier.h
+            - include/ucxx/request.h
+            - include/ucxx/request_helper.h
+            - include/ucxx/request_stream.h
+            - include/ucxx/request_tag.h
+            - include/ucxx/request_tag_multi.h
+            - include/ucxx/typedefs.h
+            - include/ucxx/utils/file_descriptor.h
+            - include/ucxx/utils/python.h
+            - include/ucxx/utils/sockaddr.h
+            - include/ucxx/utils/ucx.h
+            - include/ucxx/worker.h
+            - include/ucxx/worker_progress_thread.h
+            - lib/libucxx.so
+    about:
+      homepage: ${{ load_from_file("python/libucxx/pyproject.toml").project.urls.Homepage }}
+      license: ${{ load_from_file("python/libucxx/pyproject.toml").project.license.text }}
+      summary: ${{ load_from_file("python/libucxx/pyproject.toml").project.description }}
+
+  - package:
+      name: libucxx-examples
+      version: ${{ version }}
+    build:
+      string: ${{ date_string }}_${{ head_rev }}
+      script: cmake --install cpp/build --component examples
+      dynamic_linking:
+        overlinking_behavior: "error"
+    requirements:
+      build:
+        - ${{ compiler("c") }}
+        - ${{ compiler("cxx") }}
+        - ${{ compiler("cuda") }}
+        - cmake ${{ cmake_version }}
+      host:
+        - libucxx
```

Maybe pin the subpackage here?
```suggestion
        - ${{ pin_subpackage("libucxx", exact=True) }}
```